internal/clients/vault_secrets.go

```diff
@@ -151,3 +169,16 @@ func DeleteVaultSecretsAppSecret(ctx context.Context, client *Client, loc *share
 
 	return nil
 }
+
+func getAPIBackoffDuration(serviceErr *secret_service.OpenAppSecretDefault) time.Duration {
+	re := regexp.MustCompile(`try again in (\d+) seconds`)
+	match := re.FindStringSubmatch(serviceErr.Error())
+	baseDelayUnit := 60
```

Comment:
[Nit] Would something like backoffSeconds be a better name here? Then backoffSecondsOverride could be the name for the override value.